src/Entity/Participation.php

```diff
@@ -27,15 +27,15 @@ class Participation
 
     /**
      * @ORM\ManyToOne(targetEntity="App\Entity\Conference", inversedBy="participations", cascade={"persist"})
-     * @ORM\JoinColumn(nullable=false)
+     * @ORM\JoinColumn(nullable=true)
      */
-    private $conference;
+    private ?Conference $conference;
```

Comment:
It does not make sens to me. Why would we want to remove a Conference where someone attended?